api/integration/console_test.go

```diff
@@ -0,0 +1,70 @@
+package integration
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/replicatedhq/embedded-cluster/api"
+	"github.com/replicatedhq/embedded-cluster/api/controllers/console"
+	"github.com/replicatedhq/embedded-cluster/api/controllers/install"
+	"github.com/replicatedhq/embedded-cluster/api/pkg/installation"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConsoleListAvailableNetworkInterfaces(t *testing.T) {
```

Comment:
We should add tests for:
- unauthorized access (invalid bearer token provided)
- `a.consoleController.ListAvailableNetworkInterfaces()` returns an error